docs/fixers.rst

```diff
@@ -223,70 +223,70 @@ version of ``six`` is installed.
    Care is taken not to call ``list()`` when ``range()`` is used in an iterating
    context.
 
-.. 2to3fixer:: zip
+.. fissix_fixer:: zip
 
    If :func:`zip <python2:zip>` is called, ``from six.moves import zip`` is added to the module.
    Wrapping the use in a call to ``list()`` is done when necessary.
 
 
-``2to3`` fixers
+``fissix`` fixers
 +++++++++++++++
 
 Some `fixers from fissix <https://docs.python.org/3/library/2to3.html#fixers>`_
```

Comment:
@jreese I'd like to reference the fissix docs here https://github.com/jreese/fissix/issues/23